app/adapters/activity.js

```diff
@@ -0,0 +1,69 @@
+import Ember from 'ember';
+import DS from 'ember-data';
+
+export default DS.RESTAdapter.extend({
+    findAll: function(){
+        return new Ember.RSVP.Promise(function (resolve) {
+            let data = JSON.parse(window.localStorage.getItem('activities'));
+            if(!data || data == "") {
+                data = {activities:[]};
+            }
+            resolve(data);
+        });   
+    },
+    updateRecord: function(store, type, snapshot){
+        let currentActivities = JSON.parse(window.localStorage.getItem('activities'));
+        let record = store.serializerFor(type.modelName).serialize(snapshot);
+        record.id = snapshot.record.id;
+        if(currentActivities && currentActivities.activities && currentActivities.activities){
```

Comment:
Is "currentActivities.activities && currentActivities.activities" intentional?

Reply:
I don't think so... I'll remove the duplicate checks.